templates/issues.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-collect",
+    "exclude_completeness": [
+      "{{@ system @}}-{{@ datatype @}}-share"
+    ],
+    "namespaced_identifiers": false,
+    "source": {
+      "is_since_comparable": true,
+      "operation": "{{@ datatype @}}-list",
+      "supports_since": true,
```

Comment:
I think where we landed yesterday is that you track doesn't support since? At least not easily